packages/paste-website/src/constants.ts

```diff
@@ -46,6 +46,7 @@ export const SidebarCategoryRoutes = {
   ACCESSIBILITY: '/accessibility',
   FOUNDATIONS: '/foundations',
   CONTENT: '/foundations/content',
+  FOUNDATIONS_LOCALIZATION: '/foundations/localization',
```

Comment:
Suggestion (non-blocking): I didn't document this well, but these constants are only for disclosures in the sidebar. Since localization isn't a disclosure within foundations, you don't need to add it here.